.changeset/gold-toys-smell.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": minor
```

Comment:
If this is to land in a minor and just passes the args to c3, behaviour between c3's git init and generate needs to match 1:1. I've opened https://github.com/cloudflare/workers-sdk/pull/6415 for a bunch of differences and issues I've found with c3's implementation.

Reply:
good catch. thank you! @irvinebroque can you please address this?

Reply:
With that, I think the biggest differences will be the "non-interactive mode", but I'd recommend running through all the differences I outlined in https://github.com/cloudflare/workers-sdk/issues/6459 with various use-cases (and perhaps not removing the `generate` tests?) to ensure no breaking changes accidentally land here.

`clones a cloudflare template across drives` was a test I contributed, for example, and ensuring that doesn't regress would still be a good idea.

Reply:
The deleted tests are both related to using a named template, which is now covered in [this test](https://github.com/cloudflare/workers-sdk/pull/6415/files#diff-79dc388dc2993d1c5fa4b5c5afc48f26cdf2079685eb491e9aa931af37b4bf0cR58) as it delegates to C3.